jni/tests/faiss_wrapper_unit_test.cpp

```diff
@@ -67,20 +108,19 @@ struct MockIdMap : faiss::IndexIDMap {
     }
 };
 
-struct QueryIndexHNSWTestInput {
+struct QueryIndexInput {
     string description;
     int k;
-    int efSearch;
     int filterIdType;
     bool filterIdsPresent;
     bool parentIdsPresent;
+    int efSearch;
+    int nprobe;
 };
 
-
-
-class FaissWrappeterParametrizedTestFixture : public testing::TestWithParam<QueryIndexHNSWTestInput> {
+class FaissWrapperParametrizedTestFixture : public testing::TestWithParam<QueryIndexInput> {
```

Comment:
Nit: looks like there was an existing typo before your changes that happens a few times, should be parameterized I believe

Reply:
Corrected